src/www/ui/api/index.php

```diff
@@ -110,37 +113,37 @@ function (){
     $this->put('/{id:\\d+}', UploadController::class . ':copyUpload');
     $this->post('', UploadController::class . ':postUpload');
     $this->get('/{id:\\d+}/summary', UploadController::class . ':getUploadSummary');
-    $this->get('/{id:\\d+}/licenses', UploadController::class . ':getUploadLicenses');
+    $this->get('/{id:\\d+}/licenses', UploadController::class . ':getUploadLicenses');    
```

Comment:
Please remove unnecessary whitespace changes here and in following lines.

Reply:
> Please remove unnecessary whitespace changes here and in following lines.

Okay sir

Reply:
Please, you do not need to call me sir :smiley:

Reply:
> Please, you do not need to call me sir
Okay :sweat_smile: